src/olympia/discovery/api_urls.py

```diff
@@ -0,0 +1,14 @@
+from django.conf.urls import include, url
+
+from rest_framework.routers import SimpleRouter
+
+from . import views
+
+# Because we're registering things at "/discovery/" we don't want the trailing
+# slash that comes with the Router by default.
```

Comment:
Yeah, I don't like doing the `v3/` include because then it may overwrite anything, but you're right, that's more flexible, I'll change it.